### PR TITLE
Sankey diagram should occupy all available area (not just the left part)

### PR DESCRIPTION
**Pull request by kravets-levko**
## What type of PR is this? (check all applicable)

- [x] Bug Fix

## Description

Due to a bug in `d3-sankey` it was using wrong count of columns to compute column width, therefore not using all available area to build graph.

## Related Tickets & Documents

Closes getredash/redash#4763

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)

Before:

image

After:

image

### CODE REVIEW

File: client/app/visualizations/sankey/d3sankey.js [hunk 1]
```diff
@@ -84,17 +84,10 @@ function Sankey() {
 
     //
     moveSinksRight(x);
+    x = d3.max(nodes, n => n.x); // get new maximum x value
```
Comment (kravets-levko) [inline client/app/visualizations/sankey/d3sankey.js]: The fix :point_up:

File: client/app/visualizations/sankey/d3sankey.js [hunk 2]
```diff
@@ -84,17 +84,10 @@ function Sankey() {
...
     scaleNodeBreadths((size[0] - nodeWidth) / (x - 1));
   }
 
-  function moveSourcesRight() {
```
Comment (kravets-levko) [inline client/app/visualizations/sankey/d3sankey.js]: Unused function (dead code)

File: client/app/visualizations/sankey/d3sankey.js [hunk 3]
```diff
@@ -166,39 +159,6 @@ function Sankey() {
       });
     }
 
-    function resolveCollisions() {
-      nodesByBreadth.forEach(nodes => {
```
Comment (kravets-levko) [inline client/app/visualizations/sankey/d3sankey.js]: Duplicate code fragment (IntelliJ IDEA became smarter after update and managed to find it this time :grin:)

**Comment by arikfr** [comment]
👍

**Pull request merged**